webapp/components/emoji_picker/components/emoji_picker_preview.jsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+import EmojiStore from 'stores/emoji_store.jsx';
+
+import {FormattedMessage} from 'react-intl';
+
+export default class EmojiPickerPreview extends React.Component {
+    static propTypes = {
+        emoji: React.PropTypes.object
+    }
+
+    render() {
+        const emoji = this.props.emoji;
+
+        if (emoji) {
+            let name;
+            let aliases;
+            if (emoji.name) {
+                // This is a custom emoji that matches the model on the server
+                name = emoji.name;
+                aliases = [emoji.name];
+            } else {
+                // This is a system emoji which only has a list of aliases
+                name = emoji.aliases[0];
+                aliases = emoji.aliases;
+            }
+
+            return (
+                <div className='emoji-picker__preview'>
+                    <div className='emoji-picker__preview-image-box'>
+                        <img
+                            className='emoji-picker__preview-image'
+                            src={EmojiStore.getEmojiImageUrl(emoji)}
```

Comment:
When I hover over each emoji I have to download that emoji from the server because this is using the specific emoji URL. This isn't a great experience on a slow connection. Can we change this to pull from the sprite sheet?